django_girls_tutorial/step2/jingu/djangogirls/blog/tests.py

```diff
@@ -172,4 +208,71 @@ def test_post_update_with_error_with_post_on_404(self):
         self.assertEqual(post.text, "test text")
         response = json.loads(response.content)
         # And: 응답 메세지로 post를 찾을 수 없습니다. 를 리턴 해야 한다.
-        self.assertEqual(response["message"], "post를 찾을 수 없습니다.")
\ No newline at end of file
+        self.assertEqual(response["message"], "post를 찾을 수 없습니다.")
+
+
+class TestPostDelete(TestPostMixin, TestCase):
+    def setUp(self):
+        super().setUp()
+        self.post = Post.objects.create(
+            author=self.author, title="test title", text="test text"
+        )
```

Comment:
```suggestion
    def setUp(self):
        super().setUp()
```

```python

```
TestPostMixin의 _create_post을 쓰면 될거 같음~(중복)